app/controllers/warehouse_reports/recidivism_controller.rb

```diff
@@ -8,24 +8,20 @@ module WarehouseReports
   class RecidivismController < ApplicationController
     include WarehouseReportAuthorization
     include ArelHelper
+
     def index
-      columns = [:client_id, :project_name, :first_date_in_program, :last_date_in_program, :move_in_date, :computed_project_type, :id]
       @filter = ::Filters::DateRange.new(date_range_options)
-      ph_scope = ph_source.open_between(start_date: @filter.start, end_date: @filter.end).distinct
+      ph_scope = ph_source.joins(:project, project: [:organization]).open_between(start_date: @filter.start, end_date: @filter.end).distinct
```

Comment:
I _think_ this is the same result
```suggestion
      ph_scope = ph_source.joins(project: :organization).open_between(start_date: @filter.start, end_date: @filter.end).distinct
```

Reply:
ah yeah, that generates the same SQL. I did this in a bunch of places, I'll update them